src/features/defi/providers/yearn/components/YearnManager/Deposit/YearnDeposit.tsx

```diff
@@ -630,7 +630,14 @@ export const YearnDeposit = ({ api }: YearnDepositProps) => {
         minWidth={{ base: 'auto', lg: '450px' }}
         maxWidth={{ base: 'auto', lg: '450px' }}
       >
-        {depositRoute && <DefiActionButtons />}
+        {depositRoute && (
+          <DefiActionButtons
+            expired={
+              !!state.vault.metadata.depositsDisabled ||
```

Comment:
Do we need to coerce `state.vault.metadata.depositsDisabled`? It's already of type `boolean | undefined`.

Reply:
fixed